app/components/Exchange/View/MarketOrdersView.jsx

```diff
@@ -0,0 +1,237 @@
+import React from "react";
+import counterpart from "counterpart";
+import utils from "common/utils";
+import Translate from "react-translate-component";
+import PriceText from "../../Utility/PriceText";
+import AssetName from "../../Utility/AssetName";
+const rightAlign = {textAlign: "right"};
+import {Tooltip, Checkbox} from "bitshares-ui-style-guide";
+
+function MarketOrdersViewTableHeader({
```

Comment:
Is there a specific reason you are not using actual components here?

Reply:
In order to separate code and pure html components I decided to use stateless functions, as they don't need any of the React.Components functions. I did some reading on the topic and finally decided to do it this way. If there is even better ways to do this, I'm all ears.

Reply:
> I'm all ears

This seams like a good read https://medium.com/of-all-things-tech-progress/understanding-mvc-architecture-with-react-6cd38e91fefd#apply-mvc-with-react-flux

Reply:
Found that one as well, will dig into it and read what may work for us.

Reply:
@gibbsfromncis He was all for sorting out business logic from view. Thoughts? Leading example?